server/vcs.go

```diff
@@ -20,11 +21,11 @@ func (s *Server) registerVCSRoutes(g *echo.Group) {
 			CreatorID: c.Get(getPrincipalIDContextKey()).(int),
 		}
 		if err := jsonapi.UnmarshalPayload(c.Request().Body, vcsCreate); err != nil {
-			return echo.NewHTTPError(http.StatusBadRequest, "Malformatted create VCS request").SetInternal(err)
+			return echo.NewHTTPError(http.StatusBadRequest, "Malformed create VCS request").SetInternal(err)
```

Comment:
nit: Malformed maybe a better word, but Malformatted is used across the codebase, so would be better to leave as is.

Reply:
@tianzhou I can send a followup PR to update all occurrences of "Malformatted" to "Malformed", do you think that's worth it? Meanwhile, I'll revert the change here.

Reply:
Personally, I don't see much difference between "Malformatted" and "Malformed", so I would just keep it as is to avoid messing up the revision history. Of course, I could be persuaded here.

Reply:
I have no strong preference here, just have been using "Malformed" ever since.

Here is the quick usage comparison on sourcegraph.com:

<img width="706" alt="CleanShot 2022-05-04 at 21 06 10@2x" src="https://user-images.githubusercontent.com/2946214/166687256-2fb6550f-ef98-4fd3-b02f-77106fc31f90.png">

Looks like "Malformed" is dominating the usage at least.

Reply:
SG, I am fine to go with the common wisdom.